src/Config/RuntimeConfigLoader.cs

```diff
@@ -183,21 +142,28 @@ public static bool TryParseConfigMultipleDatabase(string json, [NotNullWhen(true
 
         return true;
     }
+
     /// <summary>
     /// Get Serializer options for the config file.
     /// </summary>
     /// <param name="replaceEnvVar">Whether to replace environment variable with value or not while deserializing.
     /// By default, no replacement happens.</param>
     public static JsonSerializerOptions GetSerializationOptions(bool replaceEnvVar = false)
     {
+        List<String> propertiesToExcludeForSerialization = new()
+        {
+            nameof(RuntimeConfig.DefaultDataSourceName),
+            nameof(RuntimeConfig.DataSourceNameToDataSource),
+            nameof(RuntimeConfig.EntityNameToDataSourceName)
+        };
```

Comment:
this just essentially excludes fields users don't need to know or care about that are explicit to multi database scenario?

Reply:
Yes!